lib/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -76,8 +77,20 @@ const isEqualValueSyntaxes = (value1, value2) => {
 		return true;
 	}
 
-	const value1Node = parse(value1, { context: 'value' });
-	const value2Node = parse(value2, { context: 'value' });
+	if (!(isStandardSyntaxValue(value1) && isStandardSyntaxValue(value2))) {
+		return false;
+	}
+
+	let value1Node;
+	let value2Node;
+
+	try {
```

Comment:
Looks like more test cases are needed.

Reply:
Feel free to add cases. 👌🏼

Reply:
The added case is handled at the line of:

```js
if (!(isStandardSyntaxValue(value1) && isStandardSyntaxValue(value2))) {
```

so finding a test case causing a syntax error may be hard. I think the current case is enough, but you may add more cases if possible. 👍🏼

Reply:
Test case added.